it/utils/IntegrationTest.scala

```diff
@@ -49,20 +46,19 @@ import views.html.templates.AgentAuthErrorPageView
 import scala.concurrent.duration.Duration
 import scala.concurrent.{Await, Awaitable, ExecutionContext, Future}
 
-// scalastyle:off number.of.methods
-// scalastyle:off number.of.types
 trait IntegrationTest extends AnyWordSpec with Matchers with GuiceOneServerPerSuite with WireMockHelper
   with WiremockStubHelpers with BeforeAndAfterAll {
+
   val nino = "AA123456A"
   val mtditid = "1234567890"
   val sessionId = "sessionId-eb3158c2-0aff-4ce8-8d1b-f2208ace52fe"
   val affinityGroup = "affinityGroup"
   val taxYear = 2022
-
-  val xSessionId: (String, String) = "X-Session-ID" -> sessionId
+  val defaultUser: EmploymentUserData = anEmploymentUserData
```

Comment:
do we still need the above variables, nino/mtditid/sessionId if they are all the same as what is in defaultUser? could we not just reference defaultUser.nino etc

Reply:
No, I was thinking to remove them but forgot. I will change them for now, but at some point I will try to remove them.